app/models/test/client_request.rb

```diff
@@ -0,0 +1,23 @@
+module Test
```

Comment:
not if we are only using it for testing purposes. I would expect to see a class for testing purposes in there. It is not impossible that an app would have a use case for something called `Test` that is unrelated to specs, so I think putting it in there might be clearer.